server/src/controllers/cart.controller.ts

```diff
@@ -0,0 +1,58 @@
+import express from "express";
+import Cart from "../models/Cart/Cart";
+
+const cartController = {
+  async showCart(
+    req: express.Request,
+    res: express.Response
+  ): Promise<express.Response | void> {
+    try {
+      const userId = req.body.user._id;
+      const myCart = await Cart.findById({ userId });
```

Comment:
`findById` takes only the ID as a parameter.
```suggestion
      const myCart = await Cart.findById(userId);
```